components/profile_popover/index.js

```diff
@@ -21,6 +29,34 @@ import {getSelectedPost, getRhsState} from 'selectors/rhs';
 
 import ProfilePopover from './profile_popover.jsx';
 
+function hasManageChannelMemberPermission(state) {
+    const myMemberships = getMyChannelMemberships(state);
+    const teamId = getCurrentTeamId(state);
+    const permissions = [
+        Permissions.MANAGE_PUBLIC_CHANNEL_MEMBERS,
+        Permissions.MANAGE_PRIVATE_CHANNEL_MEMBERS,
+    ];
+
+    for (const channelId in myMemberships) {
+        if (myMemberships.hasOwnProperty(channelId)) {
+            for (const permission of permissions) {
```

Comment:
I think the for loop here for the two permissions is not needed I would use something like:
```js
if (haveIChannelPermission(state, {channel: channelId, team: teamId, Permissions.MANAGE_PUBLIC_CHANNEL_MEMBERS}) ||
    haveIChannelPermission(state, {channel: channelId, team: teamId, Permissions.MANAGE_PRIVATE_CHANNEL_MEMBERS}) {
    return true;
}
```